### PR TITLE
Add refresh on zone display mismatch

### PR DESCRIPTION
**Pull request by linknum23**
Make zones display update when they are moved from one source to another.

To test:
1. Open two instances of the AmpliPi app ( on your phone and also on your browser?)
2. Add a zone to a different source in app 1
3. Verify that app 2 updates to display the new zone

### CODE REVIEW

**Comment by Lohrer** [comment]
There's some buggery going on here and I can't find the pattern to it. In general, you get kicked over to source 0 half the time there's a zone update. Also, if you go back on say amplipi.local/0 to amplipi.local/3, it kicks you back to amplipi.local/0 after a second.

**Comment by Lohrer** [comment]
Oh also it doesn't work on the mobile app.

**Comment by linknum23** [comment]
> Oh also it doesn't work on the mobile app.

It should now with the new zeroconf interfaces

> There's some buggery going on here and I can't find the pattern to it. In general, you get kicked over to source 0 half the time there's a zone update. Also, if you go back on say amplipi.local/0 to amplipi.local/3, it kicks you back to amplipi.local/0 after a second.

Try it out now, seemed more stable in my limited testing

File: web/static/js/script.js [hunk 1]
```diff
@@ -63,6 +63,13 @@ function debounce(ms, fun){
 function refresh() {
   get();
 }
+current_src = 's0'; // this needs to be set base on the URL path
```
Comment (Lohrer) [inline web/static/js/script.js:66]:
```suggestion
current_src = 's0'; // this needs to be set based on the URL path
```

**Review by Lohrer — APPROVED** [review]
Works much nicer now, both on web and Android!

**Pull request merged**